docs/testplan/Macsec_Rekeying.md

```diff
@@ -0,0 +1,57 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+# Test name
+
+Macsec Rekeying
```

Comment:
@parmarkj Thanks for this test -- just wanted to point out there is already a test in macsec testsuite to cover rekey scenario. https://github.com/sonic-net/sonic-mgmt/blob/fa5ab2566c9e30a6cbdaab87345eadd9ae63e7b2/tests/macsec/test_controlplane.py#L65

Can you check if that test can be enhanced ?

Reply:
closing PR as test already covered